Gemfile

```diff
@@ -65,40 +77,33 @@ if puppet_gem_location != :gem || puppetversion < '3.5.0'
   if Gem::Platform.local.os == 'mingw32'
     explicitly_require_windows_gems = true
   end
-
-  if puppet_gem_location == :gem
-    # If facterversion hasn't been specified and we are
-    # looking for a Puppet Gem version less than 3.5.0, we
-    # need to ensure we get a good Facter for specs.
-    gem "facter",">= 1.6.11","<= 1.7.5",:require => false unless facterversion
-    # If hieraversion hasn't been specified and we are
-    # looking for a Puppet Gem version less than 3.5.0, we
-    # need to ensure we get a good Hiera for specs.
-    gem "hiera",">= 1.0.0","<= 1.3.0",:require => false unless hieraversion
-  end
```

Comment:
And here is why it started breaking. Looks like we have facter version fixed but not hiera version.